### PR TITLE
Drawing App

### PR DESCRIPTION
**Pull request by Sulagna-Dutta-Roy**
## Related Issue
  - Info about Issue or bug
  I've added the drawing app as an additional project using Javascript.

Closes: #[issue number that will be closed through this PR]
#1908 

### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!
I've added all the files in this project.The project is drawing app using Javascript. There is an small canvas that let user can do anything they want. The user can draw and erase their painting.Reset option also provided.The slider can increase or decrease the paint color fill.

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.
NA
### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.
Drawing
Drawing1

### CODE REVIEW

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Hello! I think you forgot to add the project details in script.js file. The changes won't be reflected until you add your project details there.

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Hello! I think you have misunderstood the request. You need to add the project details to script.js file. 
Here is the link to the file which I am talking about: [https://github.com/Vishal-raj-1/Awesome-JavaScript-Projects/blob/main/script.js](https://github.com/Vishal-raj-1/Awesome-JavaScript-Projects/blob/main/script.js) .
Also, the css and js files needs to be in assets folder in their respective directories.

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Everything is good. However, you still need to move the js and css files in assets folder.

**Comment by Sulagna-Dutta-Roy** [comment]
I've done the changes. Please check sir

File: public/Drawing App/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="initial-scale=1.0,user-scalable=yes">
+        <title>Drawing App</title>   
+        <link rel="stylesheet" href="style.css">
```
Comment (Rizwan-S) [inline public/Drawing App/index.html:7]: should  be 
```
<link rel="stylesheet" href="../../assets/css/Drawing App/style.css">
```

File: public/Drawing App/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                <span id="reset" class="button">Reset</span>
+            </div>
+            <!-------------------------->
+            <div class="input2">
+                <span id="save" class="button">Save</span>
+            </div>
+        </div>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
+        <script src="script.js"></script>
```
Comment (Rizwan-S) [inline public/Drawing App/index.html:49]: Should be
```
<script src="../../assets/js/Drawing App/script.js"></script>
```

**Review by Rizwan-S — CHANGES_REQUESTED** [review]

**Comment by Sulagna-Dutta-Roy** [comment]
Done

**Review by Rizwan-S — APPROVED** [review]

**Review by Vishal-raj-1 — APPROVED** [review]

**Pull request merged**